src/components/Documentation/Layout/SidebarMenu/index.tsx

```diff
@@ -51,37 +67,34 @@ function SidebarMenuItem({ children, label, path, activePaths, onClick }) {
   )
 }
 
-SidebarMenuItem.propTypes = {
-  children: PropTypes.arrayOf(PropTypes.object),
-  label: PropTypes.string.isRequired,
-  path: PropTypes.string.isRequired,
-  source: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]).isRequired,
-  onClick: PropTypes.func,
-  activePaths: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.string),
-    PropTypes.bool
-  ]).isRequired
+interface ISidebarMenuProps {
+  currentPath: string
+  onClick: (e: React.MouseEvent) => void
 }
 
-export default function SidebarMenu({ id, sidebar, currentPath, onClick }) {
-  const psRef = useRef()
+const SidebarMenu: React.SFC<ISidebarMenuProps> = ({
+  currentPath,
+  onClick
+}) => {
+  const rootRef = useRef<HTMLDivElement>(null)
+  const psRef = useRef<PerfectScrollbar | undefined>(undefined)
   const [isScrollHidden, setIsScrollHidden] = useState(false)
   const activePaths = currentPath && getParentsListFromPath(currentPath)
 
   useEffect(() => {
-    if (!psRef.current) {
-      psRef.current = new PerfectScrollbar(`#${id}`, {
+    if (!psRef.current && rootRef.current) {
+      psRef.current = new PerfectScrollbar(rootRef.current, {
         wheelPropagation: true
       })
     }
 
-    const node = document.getElementById(currentPath)
-    const parent = document.getElementById(id)
+    const node = document.getElementById(currentPath.replace(/\/$/, ''))
```

Comment:
looks like a hack ... why is it needed? is there way to do this properly?

Reply:
Yes, my bad. It was necessary for testing it in the development mode. Removed.